extensions/data-transfer/portability-data-transfer-koofr/src/main/java/org/datatransferproject/transfer/koofr/photos/KoofrPhotosImporter.java

```diff
@@ -225,4 +223,20 @@ private Date getDateCreated(PhotoModel photo, byte[] bytes) {
       return null;
     }
   }
+
+  private synchronized SimpleDateFormat getOrCreateTitleDateFormat(UUID jobId) {
+    if (titleDateFormats.containsKey(jobId)) {
+      return titleDateFormats.get(jobId);
+    }
+
+    SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd HH.mm.ss ");
```

Comment:
NITPICK `"yyyy-MM-dd HH.mm.ss "` could be moved to a class constant.

Reply:
Done!